README.md

```diff
@@ -1,3 +1,19 @@
 # Paketo Git Buildpack
+The Paketo Git Buildpack is a Cloud Native Buildpack that retreives `git` metadata and preforms `git` operations.
 
-Implementing [RFC #0023](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0023-git-buildpack.md) and resolving [#45](https://github.com/paketo-buildpacks/rfcs/issues/45).
+## Behavior
+This buildpack uses the `git` dependency off of the stack that it is running on top of. The Git buildpack will only participate if there is a valid `.git` directory in the application source directory.
+
+The buildpack will do the following:
+
+- Sets the `REVISION` environment variable, which is the commitish of HEAD, to be available for the build processes of other buildpacks and in the final running image.
```

Comment:
It also sets the `org.opencontainers.image.revision` label with that same commitish: https://github.com/paketo-community/git/blob/d413da8996789d0dc50078da1077acf611f085c1/build.go#L68